test/scheduler/ExecuatbleCronSchedule.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { DateTime } from 'luxon';
+import { CronSchedule } from '../../src/job/MomoJob';
+import { ExecutableCronSchedule } from '../../src/scheduler/ExecutableCronSchedule';
+
+describe('ExecutableIntervalSchedule', () => {
+  const callbackFunction = jest.fn();
+
+  let cronSchedule: CronSchedule;
+  let executableCronSchedule: ExecutableCronSchedule;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
you don't actually do anythin with the mocked function. Maybe just use a noop for callbackFunction and remove this line